.github/workflows/sync-issue-with-pr.yaml

```
@@ -0,0 +1,46 @@
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Get Linked Issues
+        id: get_issues
+        run: |
+          # Find issue numbers linked with PR using keywords like Fixes, Closes
+          ISSUE_NUMBERS=$(gh pr view ${{ github.event.pull_request.number }} --json closingIssuesReferences --jq '.closingIssuesReferences[].number')
+          echo "Linked issues: $ISSUE_NUMBERS"
+          echo "::set-output name=issues::$ISSUE_NUMBERS"
+      - name: Read Issue Labels and Titles
+        id: issue_data
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2043:warning:2:21: This loop will only ever run once. Bad quoting or missing glob/expansion?

(shellcheck)